### DISCUSSION TITLE
Is possible to get the products sales data from tiktok shop using this unofficial API?

### DISCUSSION THREAD

**Discussion by mono300genuine** [davidteather/TikTok-Api#1189] [General]
Hello, everyone! 

I am happy to connect with you.

I am planning to build tool that can get the products sales data from tiktok shop.

Can I get sales data from TikTok Shop using the unofficial API of this repository? 

For example, seller name, product ID, inventory, sales, reviews, etc. 

I also want to know if there are any country restrictions on using this API.

Welocom to any comments and advice.

Thank you.

**Comment by zakrian07** [comment]
Im working on it too scrap mobile app if @davidteather/TikTok-Api
***@***.***>  is interested we can make a feature branch
